docs/features/tenancy.md

```diff
@@ -1,8 +1,9 @@
-* IMPORTANT: This is an experimental feature supported only on standalone ESX as of release 0.12. We are seeking feedback/requirements to support mult-tenancy across vSphere Cluster/Datacenter. You can drop us a note at containers@vmware.com.
+---
```

Comment:
sorry .. can you elaborate here? what do you mean by "Multi-tenancy is not a experimental feature any more?"